ui/lib/core/addon/components/radio-select-ttl-or-string.hbs

```diff
@@ -0,0 +1,50 @@
+<div class="column is-narrow is-flex-center has-text-grey has-right-margin-s has-top-margin-negative-s">
+  <RadioButton
+    class="radio"
+    name={{@attr.name}}
+    id={{"ttl"}}
+    @value={{"ttl"}}
+    @onChange={{this.selectionChange}}
+    @groupValue={{this.groupValue}}
+  />
+  <label class="has-left-margin-xs">
+    <TtlPicker2
+      data-test-input={{@attr.name}}
+      @onChange={{this.setAndBroadcastTtl}}
+      @label={{"TTL"}}
+      @helperTextEnabled={{@attr.options.helperTextEnabled}}
+      @description={{@attr.helpText}}
+      @time={{this.ttlTime}}
+      @unit="d"
+      @hideToggle={{true}}
+    />
+  </label>
+</div>
+<div class="column is-narrow is-flex-center has-text-grey has-right-margin-s">
+  <RadioButton
+    class="radio"
+    name={{@attr.name}}
+    id={{"specificDate"}}
+    @value={{"specificDate"}}
+    @onChange={{this.selectionChange}}
+    @groupValue={{this.groupValue}}
+  />
+  <label class="has-left-margin-xs">
+    <span class="ttl-picker-label is-large">Specific date</span><br />
+    <p class="sub-text">
+      This value format should be given in UTC format YYYY-MM-ddTHH:MM:SSZ.
+    </p>
+    {{#if (eq this.groupValue "specificDate")}}
+      <input
+        data-test-input={{@attr.name}}
+        id={{@attr.name}}
```

Comment:
If you're going to use this component elsewhere in PKI will it be to set the same params of `not_after` and `ttl`? If that's the case - what about making those the ids here?

Another possibility, to make this more dynamic, would be to add them as options for the `customTtl` attr on the model 
```
  @attr({
    label: 'Not valid after',
    editType: 'yield',
    modelAttrs: {
      ttl: 'ttl',
      input: 'not_after',
    },
  })
  customTtl;
})
``` 

and then make the id here ` id={{@attr.options.modelAttrs.input}}` which would allow directly passing `not_after` or `ttl` to the onChange

Happy to explain better/pair if you want to go that direction

Reply:
Yeah, I like that direction. The same pattern is going to be used for all instances of this (e.g. the same form fields) so I think without having to remember to add modelAttrs to the models, I'll go ahead and hard code, which will also tell a future developer what these fields are for.